<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>time_set_win</class>
 <widget class="QDialog" name="time_set_win">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="sld_firsttime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="value">
    <number>9</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sld_secondtime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="value">
    <number>23</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_firsttime">
   <property name="geometry">
    <rect>
     <x>-210</x>
     <y>10</y>
     <width>371</width>
     <height>120</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>9</number>
   </property>
   <property name="value" stdset="0">
    <double>9.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_secondtime">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>10</y>
     <width>391</width>
     <height>120</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>9</number>
   </property>
   <property name="value" stdset="0">
    <double>23.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>41</width>
     <height>181</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>О
к
е
й</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
